.github/workflows/cron-blue-green-deploy.yml

```diff
@@ -57,22 +59,28 @@ jobs:
         run: pnpm install --no-frozen-lockfile
       - name: Install Vercel CLI
         run: npm install --global vercel
+        # Pull down the production environment variables as if were building on Vercel.
+        # https://vercel.com/docs/cli/pull
       - name: Get Vercel Environment Variables
         run: vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
+        # Create a "blue" version of the site, deloy it, adnd capture the deployment url.
       - name: Update CSS to Blue
         run: 'echo "body {background: blue;}" > ./app/blue-green.css'
       - name: Build Blue
         run: vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
       - name: Deploy Blue
         id: deploy-blue
         run: echo "url=$(vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }})" >> $GITHUB_OUTPUT
+        # Create a "green" version of the site, deloy it, adnd capture the deployment url.
```

Comment:
```suggestion
        # Create a "blue" version of the site, deploy it, and capture the deployment url.
```

Reply:
Addressed in https://github.com/vercel-labs/blue-green/commit/2f96c2cb3123bd9fb29565886a024eddbfb095be.